src/libs/utils.js

```diff
@@ -421,6 +421,31 @@ export const getSearchFilterFunctions = () => {
         includes(loweredTerm, toLower(datasetTerm.study?.studyName)) ||
         includes(loweredTerm, toLower(status));
     }, targetList),
+    institutions: (term, targetList) => filter(institution => {
+      const loweredTerm = toLower(term);
+      const soStrings = institution.signingOfficials?.map((so) => {
+        return so.displayName + ' ' + so.email;
+      }).join(' ');
+      const domains = institution.domains?.join(' ') || '';
+      return includes(loweredTerm, toLower(institution.name)) ||
+        includes(loweredTerm, toLower(institution.id)) ||
```

Comment:
Am I understanding that we'll call toLower on each cell in the table, each time the filter changes?

Reply:
Correct. You can see how we're using the same pattern in all of the other filter methods in this utility.

Reply:
A future improvement could be to cache a copy of the data once lower cased if we need to deal with lower power devices.

Reply:
See https://broadworkbench.atlassian.net/browse/DT-2013